Makefile

```diff
@@ -0,0 +1,11 @@
+bootstrap:
```

Comment:
Wonder if adding "cleanup" command would be a good idea? It would tear down the db and its volume.

Reply:
As discussed, @krzysztofwolski will open separate issue for it

Reply:
#159 task added 👍